TYPO3.TypoScript/Classes/TYPO3/TypoScript/Core/Cache/RuntimeContentCache.php

```diff
@@ -65,6 +71,31 @@ public function __construct(Runtime $runtime)
     }
 
     /**
+     * @param string $key
+     * @param string $value
+     */
+    public function addTag($key, $value)
+    {
+        $key = trim($key);
+        $value = trim($value);
+        if ($key === '' || $value === '') {
+            return;
+        }
+        $tag = Functions::ucfirst($key) . 'DynamicTag_' . $value;
+        $this->tags[$tag] = true;
+    }
+
+    /**
+     * @return array
+     */
+    protected function flushTag()
+    {
+        $tags = array_keys(array_filter($this->tags));
```

Comment:
a small comment would be good what array_filter is supposed to do (strip FALSE values; but how can they happen?)